### DISCUSSION TITLE
[1.16] Option to lock plant sower's input slots

### DISCUSSION THREAD

**Discussion by Based199** [InnovativeOnlineIndustries/Industrial-Foregoing#897] [Completed/Done]
**Is your feature request related to a problem? Please describe.**

So I've been having an issue with variety and the plant sower. it plants really well, and it's super cool to have 9 different plants being planted at the same time, however refilling the seeds is the problem. One type of seed, say wheat seeds, are produced in excess, which will then fill up the inventory. This then prevents other seeds from being inserted. 

**Describe the solution you'd like**

Basically, a button that locks the seeds that are currently in the plant sower. Which will allow only the selected seed in the applicable inventory slot.

**Additional context**
